test/fixtures/go-key-ed25519.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+module.exports = {
```

Comment:
Could leave a comment with the commands you used to generate these? (I forgot to do that with the other ones, and missed it once bitterly)